docs/ocs-api-swagger.json

```diff
@@ -1,372 +1,415 @@
 {
-  "swagger": "2.0",
-  "info": {
-    "description": "[Intel FDO](https://software.intel.com/en-us/secure-device-onboard) (FIDO Device Onboard) is a technology that is created by Intel to make it easy and secure to configure edge devices and associate them with an Open Horizon Management Hub instance. Open Horizon has added support for FDO-enabled devices so that the agent will be installed on the device and registered to the Open Horizon Management Hub with zero touch (by simply powering on the device).<br><br>Examples of using this API:<br><br>`curl -sS $HZN_FDO_SVC_URL/version && echo`<br>`curl -sS -w %{http_code} -u $HZN_ORG_ID/$HZN_EXCHANGE_USER_AUTH $HZN_FDO_SVC_URL/orgs/$HZN_ORG_ID/fdo/vouchers | jq`<br><br>Note: Some of these APIs can also be run via the `hzn` command.",
-    "version": "1.2.0",
-    "title": "Open Horizon Support for FDO",
-    "license": {
-      "name": "Apache 2.0",
-      "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
-    }
-  },
-  "host": "open-horizon-mgmt-hub",
-  "basePath": "/api",
-  "tags": [
-    {
-      "name": "version",
-      "description": "Get version number"
-    },
-    {
-      "name": "vouchers",
-      "description": "Manage device ownership vouchers"
-    },
-    {
-      "name": "keys",
-      "description": "Manage device owner keys"
+    "openapi": "3.0.1",
+    "info": {
+        "title": "Open Horizon Support for FDO",
+        "description": "[FDO](https://software.intel.com/en-us/secure-device-onboard) (FIDO Device Onboard) is a technology that is contributed by Intel to make it easy and secure to configure edge devices and associate them with an Open Horizon Management Hub instance. Open Horizon has added support for FDO-enabled devices so that the agent will be installed on the device and registered to the Open Horizon Management Hub with zero touch (by simply powering on the device).<br><br>Examples of using this API:<br><br>`curl -sS $HZN_FDO_SVC_URL/api/version && echo`<br>`curl -sS -w %{http_code} -u $HZN_EXCHANGE_USER_AUTH $HZN_FDO_SVC_URL/api/orgs/$HZN_ORG_ID/fdo/vouchers | jq`<br><br>Note: Some of these APIs can also be run via the `hzn` command.",
```

Comment:
A better URL link about FDO would be https://fidoalliance.org/device-onboarding-overview/

Reply:
Fixed.